extension/package.json

```diff
@@ -331,6 +331,16 @@
       {
         "command": "gradle.hideStoppedDaemons",
         "title": "Hide Stopped Daemons"
+      },
+      {
+        "command": "gradle.createProject",
+        "category": "Gradle",
+        "title": "Create a Gradle Java Project..."
+      },
+      {
+        "command": "gradle.createProjectAdvanced",
+        "category": "Gradle",
+        "title": "Create a Gradle Java Project... (Advanced)"
```

Comment:
In long term, you can consider use a custom button to "switch to advanced mode" in the wizard. Then you only need one command as the only entry.

Reply:
Great. #1153 will track it.